front/app/containers/IdeasNewPage/WithJSONForm/index.tsx

```diff
@@ -152,9 +153,30 @@ const IdeasNewPageWithJSONForm = ({ params }: WithRouterProps) => {
     [uiSchema]
   );
 
+  // get participation method config
+  const phaseFromUrl = usePhase(phaseId);
+  let config;
+
+  if (!isNilOrError(phaseFromUrl)) {
+    config = getMethodConfig(phaseFromUrl.attributes.participation_method);
+  } else {
+    if (phases && project?.attributes.process_type === 'timeline') {
+      const participationMethod =
+        getCurrentPhase(phases)?.attributes.participation_method;
+      if (!isNilOrError(participationMethod)) {
+        config = getMethodConfig(participationMethod);
+      }
+    } else if (!isNilOrError(project)) {
+      config = getMethodConfig(project.attributes.participation_method);
+    }
+  }
```

Comment:
Should I make these conditionals into a function in the participation method utils maybe? Which takes in a project, any phases, and any phaseId from URL?